package.json

```diff
@@ -74,15 +73,15 @@
     "node-sass": "^4.14.1",
     "prettier": "^2.0.5",
     "puppeteer": "^2.0.0",
-    "raw-loader": "^3.1.0",
+    "raw-loader": "^4.0.2",
     "sass-loader": "^8.0.0",
     "shx": "^0.3.2",
     "ts-jest": "^26.1.4",
-    "tslint": "^5.20.0",
+    "ts-loader": "^8.0.17",
```

Comment:
Switching from `awesome-typescript-loader` to `ts-loader` since the former is no longer supported.